dll/win32/shell32/CDefView.cpp

```diff
@@ -1475,6 +1756,105 @@ HRESULT CDefView::CheckViewMode(HMENU hmenuView)
     return S_OK;
 }
 
+LRESULT CDefView::DoColumnContextMenu(LPARAM lParam)
+{
+    const UINT maxItems = 15; // Feels about right
+    const UINT idMore = 0x1337;
```

Comment:
Why these hardcoded values?

Reply:
I don't remember the details anymore but I believe Windows also limits the number of items it will display in this menu. Just like the comment says, 15 feels like the maximum that looks decent. If you want a less used column, you would have to click the "more" item (that is in another PR and is not going to do much until IShellFolder2 is fully implemented).

Reply:
What's idMore then I mean it's 0x1337 so maybe you chose that on purpose lol

Reply:
Good enough for me.